### ISSUE TITLE
Allow overriding scrollPadding for a single step

### ISSUE THREAD

**Issue by r-brown** [Labs64/GuideChimp#85]
Allow overriding scrollPadding for a single step
e.g.
```
    var tour = [
        {
            element: '#id1',
            title: 'Step 1',
        },
        {
            element: '#id2',
            // overrides global scrollPadding (dafault: 10)
            scrollPadding: 200,
            title: 'Step 2',
        },
        {
            element: '#id3',
            title: 'Step 3',
        },
    ];

    var guideChimp = GuideChimp(tour, { position: 'right' });
    guideChimp.start();

```

**Comment by r-brown** [comment]
fixes #84

**Comment by v-rudkovskiy** [comment]
PR: https://github.com/Labs64/GuideChimp/pull/87
